pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -291,6 +290,10 @@ func (b *awsElasticBlockStoreBuilder) SetUpAt(dir string) error {
 		return err
 	}
 
+	if !b.readOnly {
```

Comment:
@thockin I don't think we settled on the impl detail of whether you make this call if fsGroup == 0, thoughts?